examples/centralised-logging/full-setup-existing-kubeflow.sh

```diff
@@ -17,12 +17,12 @@ kubectl rollout status -n seldon-system statefulset/seldon-operator-controller-m
 
 sleep 5
 
-helm install --name seldon-single-model ../../helm-charts/seldon-single-model/ --set engine.env.LOG_MESSAGES_EXTERNALLY="true" --set model.annotations."seldon\.io/istio-gateway"="kubeflow-gateway.kubeflow.svc.cluster.local"
+helm install --name seldon-single-model --namespace default ../../helm-charts/seldon-single-model/ --set engine.env.LOG_MESSAGES_EXTERNALLY="true" --set model.annotations."seldon\.io/istio-gateway"="kubeflow-gateway.kubeflow.svc.cluster.local"
```

Comment:
Not sure why this is showing as this already seems to be in master - https://github.com/SeldonIO/seldon-core/blob/master/examples/centralised-logging/full-setup-existing-kubeflow.sh#L20